cli/src/create_organization.rs

```diff
@@ -74,9 +69,9 @@ pub async fn create_organization_req(
 pub async fn create_organization(create_organization: CreateOrganization) -> anyhow::Result<()> {
     let CreateOrganization {
         organization_id,
-        addr,
-        token,
+        server: ServerSharedOpts { addr, token },
     } = create_organization;
+    log::trace!("Creating organization \"{organization_id}\" (addr={addr})");
```

Comment:
To remain consistent with other messages below:
```suggestion
    log::trace!("Creating organization {organization_id} (addr={addr})");
```